tests/e2e/disableagent/Vagrantfile

```
@@ -0,0 +1,77 @@
+ENV['VAGRANT_NO_PARALLEL'] = 'no'
+NODE_ROLES = (ENV['E2E_NODE_ROLES'] ||
+  ["server-0", "agent-0"])
+NODE_BOXES = (ENV['E2E_NODE_BOXES'] ||
+  ['generic/ubuntu2204', 'generic/ubuntu2204', 'generic/ubuntu2204', 'generic/ubuntu2204'])
```

```suggestion
  ['generic/ubuntu2204', 'generic/ubuntu2204'])
```

I would prefer to integrate into the existing startup e2e test, but if you want to keep it standalone, this will need to be fixed.

---

Done